pkgs/os-specific/linux/firmware/fwupd/default.nix

```diff
@@ -122,18 +123,18 @@ let
 in
 stdenv.mkDerivation (finalAttrs: {
   pname = "fwupd";
-  version = "1.8.15";
+  version = "1.9.2";
 
   # libfwupd goes to lib
   # daemon, plug-ins and libfwupdplugin go to out
   # CLI programs go to out
   outputs = [ "out" "lib" "dev" "devdoc" "man" "installedTests" ];
 
   src = fetchFromGitHub {
-    owner = "fwupd";
-    repo = "fwupd";
+    owner = finalAttrs.pname;
```

Comment:
Please refer this hunk. Is pname gets appended then the fod can no longer be build.

Reply:
Right, I reverted this part, thanks.